src/prng/generator.lisp

```diff
@@ -53,11 +51,11 @@
        do (incf counter)
        finally (return (apply #'concatenate 'simple-octet-vector blocks)))))
 
-(defun pseudo-random-data (generator num-bytes)
+(defmethod prng-random-data (num-bytes (generator fortuna-generator))
```

Comment:
`PSEUDO-RANDOM-DATA` & `PRNG-RANDOM-DATA` were essentially the same function with different argument order.